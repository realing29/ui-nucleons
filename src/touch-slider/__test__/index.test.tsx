import React from 'react';
import { mount } from 'enzyme';
import TouchSlider from '..';

describe('<TouchSlider />', () => {
  it('should renders correctly', () => {
    const wrapper = mount(
      <TouchSlider>
        <div>Foo</div>
        <div>Bar</div>
        <div>Baz</div>
      </TouchSlider>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
