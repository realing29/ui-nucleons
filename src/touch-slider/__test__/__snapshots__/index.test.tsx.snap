// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TouchSlider /> should renders correctly 1`] = `
<div>
  <div
    class="layout mobile "
  >
    <div
      class="outer"
      data-bsl-ignore="true"
    >
      <div
        class="inner"
      >
        <div>
          Foo
        </div>
        <div>
          Bar
        </div>
        <div>
          Baz
        </div>
      </div>
    </div>
  </div>
</div>
`;
